bot/utils/log.py

```diff
@@ -20,3 +21,7 @@ async def log_system_message(level: int, message: str, logger: logging.Logger) -
 async def log_user_activity(user_id: int, message: str, logger: logging.Logger) -> None:
     await log_system_message(logging.INFO, message, logger)
     await DatabaseManager.log_user_activity(user_id, message)
+
+def get_log_level(env_var='LOG_LEVEL', default='INFO'):
+    log_level_str = os.getenv(env_var, default).upper()
+    return getattr(logging, log_level_str, logging.INFO)
```

Comment:
```suggestion
    return getattr(logging, log_level_str)
```

na pewno nie chcemy tutaj domyślnej wartości, jak jest podana i nie działa to ma się wywalić, bo zły konfig